### DISCUSSION TITLE
Welcome to FirstCyclingAPI Discussions!

### DISCUSSION THREAD

**Discussion by baronet2** [baronet2/FirstCyclingAPI#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jdarras** [comment]
Hello, 
My Name is Julien Darras, I'm from France, near Paris and I LOVE cyclism. I did races and cyclotourism. I'm working with Unreal Engine and try to do a project with this data and UE5.
